src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -798,6 +803,16 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                                                               falseReg, opt);
                                 break;
 
+                            case NI_Sve2_ConvertToSingleOdd:
+                            case NI_Sve2_ConvertToSingleOddRoundToOdd:
+                                // TODO-SVE: Optimise away the explicit copying of `embMaskOp1Reg` to `targetReg`.
+                                // For these intrinsics we cannot use movprfx instruction to populate `targetReg` with
+                                // `embMaskOp1Reg`. Thus, we need to perform move before the operation.
```

Comment:
Is there a tracking issue for this `TODO` already?

Reply:
Created #120939 now.